crates/sqllib/src/dec.rs

```diff
@@ -629,7 +634,7 @@ impl AddAssign<SqlDecimal> for SqlDecimal {
         let mut result = self.value;
         context.add(&mut result, &other.value);
         if is_error(&context) {
-            panic!("Error during DECIMAL computation: ADD");
+            panic!("Error during computation: {} + {}", self, other);
```

Comment:
```suggestion
            panic!("Error during computation: {} += {}", self, other);
```

Reply:
there is no += in SQL, the error won't make sense to users